src/docfx/build/legacy/metadata/LegacyMetadata.cs

```diff
@@ -111,6 +111,11 @@ public static JObject GenerateLegacyRedirectionRawMetadata(Docset docset, PageMo
             if (pageModel.UpdatedAt != default)
                 rawMetadata["updated_at"] = pageModel.UpdatedAt.ToString("yyyy-MM-dd hh:mm tt", docset.Culture);
 
+            if (pageModel.Bilingual)
+            {
+                rawMetadata["bilingual_type"] = "hover over";
```

Comment:
https://ceapex.visualstudio.com/Engineering/_workitems/edit/20449

Do we want to be consistent with template and loc team and change the term `bilingual` to `enable_loc_sxs` ?

Reply:
i don't want to it right now since it causes big impact changes right now, i will log a issue for that

Reply:
[https://github.com/dotnet/docfx/issues/3590](https://github.com/dotnet/docfx/issues/3590)

---
In reply to: [231770644](https://github.com/dotnet/docfx/pull/3588#discussion_r231770644) [](ancestors = 231770644)